lib/Conversion/PolyToStandard/PolyToStandard.cpp

```diff
@@ -721,7 +726,8 @@ void PolyToStandard::runOnOperation() {
 
   // target.addIllegalDialect<PolyDialect>();
   target.addIllegalOp<FromTensorOp, ToTensorOp, AddOp, SubOp, LeadingTermOp,
-                      MonomialOp, MonomialMulOp, ConstantOp, MulOp>();
+                      MonomialOp, MulScalarOp, MonomialMulOp, ConstantOp,
```

Comment:
This was the big problem: it wasn't in the illegal ops list, so it wasn't being lowered >_<